ui/raidboss/data/04-sb/trial/lakshmi-ex.js

```diff
@@ -50,36 +48,7 @@
       regexJa: Regexes.startsUsing({ id: '215E', source: 'ラクシュミ' }),
       regexCn: Regexes.startsUsing({ id: '215E', source: '吉祥天女' }),
       regexKo: Regexes.startsUsing({ id: '215E', source: '락슈미' }),
-      alarmText: function(data, matches) {
-        if (data.role != 'tank' && matches.target == data.me) {
-          return {
-            en: 'Buster on YOU',
-            de: 'Tankbuster auf DIR',
-          };
-        }
-      },
-      alertText: function(data, matches) {
-        if (data.role == 'tank' && matches.target == data.me) {
-          return {
-            en: 'Buster on YOU',
-            de: 'Tankbuster auf DIR',
-          };
-        }
-        if (data.role == 'healer' && matches.target != data.me) {
-          return {
-            en: 'Buster on ' + matches.target,
-            de: 'Tankbuster auf ' + matches.target,
-          };
-        }
-      },
-      tts: function(data) {
-        if (data.role == 'tank' || data.role == 'healer') {
-          return {
-            en: 'buster',
-            de: 'Basta',
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
This is the offtank cleave based on second hate.  Similar to o4s earthshakers, it's good to have an alarm for non-tanks.  Maybe this could just be two different triggers with conditions, one of which is tankBuster, and the other is tankBuster('alarm')?

Reply:
I mean it would not be a problem but in the original version both were alarms ^^
I tried it with an alarm for tank/healer and info to dps (based on the matching that happends in Response.tankbuster())